source/Calamari.Shared/Integration/Certificates/WindowsX509CertificateStore.cs

```diff
@@ -406,7 +390,55 @@ static IList<SafeCertContextHandle> GetCertificatesFromPfx(byte[] pfxBytes, stri
             }
         }
 
-        static void AddPrivateKeyAccessRules(ICollection<PrivateKeyAccessRule> accessRules, SafeCertContextHandle certificate)
+        static IList<Org.BouncyCastle.X509.X509Certificate> GetCertificatesToImport(byte[] pfxBytes, string? password)
```

Comment:
I think most of this diff is just GitHub getting confused about moving methods around. There shouldnt be any particular changes here.